hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java

```diff
@@ -45,45 +38,95 @@
     @JsonInclude
     protected long timestamp;
 
-    @JsonInclude(Include.NON_NULL)
-    protected Object value;
+    /** For single-value condition types. Null otherwise */
+    @JsonInclude(Include.NON_EMPTY)
+    protected String value;
 
-    @JsonInclude
-    protected Type type;
+    /** For multi-value condition types. Null otherwise. See the condition type for expected key-value information */
+    @JsonInclude(Include.NON_EMPTY)
+    protected Map<String, String> values;
```

Comment:
Minor comment. I think is just a matter of style, for me having Data.value and Data.values for single or multiple values it less clear that having just a values and perhaps a helper method for single values.